nixos/modules/services/monitoring/prometheus/alertmanager.nix

```diff
@@ -152,10 +136,8 @@ in {
         wantedBy = [ "multi-user.target" ];
         after    = [ "network.target" ];
         serviceConfig = {
-          User = cfg.user;
-          Group = cfg.group;
           Restart  = "always";
-          PrivateTmp = true;
+          DynamicUser = true;
```

Comment:
We should be able to keep PrivateTmp here I think

Reply:
We can remove it because `DynamicUser` implies `PrivateTmp`.